Fix:

error: implicit declaration of function 'jpeg_std_error' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: unknown type name 'size_t'
error: unknown type name 'FILE'
error: too few arguments to function call, single argument 'err' was not specified
error: implicit declaration of function 'TIFFOpen' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: too few arguments to function call, expected 2, have 0
--- SConstruct.orig	2021-02-02 04:46:51.000000000 -0600
+++ SConstruct	2021-02-02 04:48:13.000000000 -0600
@@ -68,9 +68,9 @@
     print "compiling for development (slower but safer)"
 
 assert conf.CheckLibWithHeader('png', 'png.h', 'C', 'png_byte;', 1),"please install: libpng12-dev"
-assert conf.CheckLibWithHeader('jpeg', 'jconfig.h', 'C', 'jpeg_std_error();', 1),"please install: libjpeg62-dev"    
+assert conf.CheckLibWithHeader('jpeg', ['stddef.h', 'stdio.h', 'jpeglib.h'], 'C', 'jpeg_std_error(0);', 1),"please install: libjpeg62-dev"
 assert conf.CheckLibWithHeader('z', 'zlib.h', 'C', 'inflate;', 1), "please install: zlib"
-assert conf.CheckLibWithHeader('tiff', 'tiff.h', 'C', 'TIFFOpen();', 1), "please install: libtiff4-dev"
+assert conf.CheckLibWithHeader('tiff', 'tiffio.h', 'C', 'TIFFOpen(0, 0);', 1), "please install: libtiff4-dev"
 
 ### check for optional parts
 
